runtime/compiler/trj9/runtime/RelocationRecord.hpp

```diff
@@ -216,6 +219,9 @@ class TR_RelocationRecord
       TR_RelocationRecordBinaryTemplate *_record;
 
       TR_RelocationRecordPrivateData _privateData;
+   private:
+      static TR_RelocationRecord * decode(TR_RelocationRecord *storage, TR_RelocationRuntime *reloRuntime, TR_RelocationTarget *reloTarget, TR_RelocationRecordBinaryTemplate *record);
+      static uint8_t getReloType(TR_RelocationTarget *reloTarget, TR_RelocationRecordBinaryTemplate *rec);
```

Comment:
getReloType() is probably better implemented as a nonvirtual instance function on the TR_RelocationRecordBinaryTemplate struct. That's somewhat out of character for those structs, but I like that better than adding this function here.

Reply:
getReloType() is used in a static function, so it has to be static as well (not an instance function). I could move it to TR_RelocationRecordBinaryTemplate, but still as a static function.